pkg/cli/options_test.go

```diff
@@ -34,6 +35,167 @@ import (
 )
 
 var _ = Describe("Discover external plugins", func() {
+	Context("with valid plugins root path", func() {
+		var (
+			homePath           string = os.Getenv("HOME")
+			customPath         string = "/tmp/myplugins"
+			xdghome            string = os.Getenv("XDG_CONFIG_HOME")
+			originalPluginPath string = os.Getenv("KB_PLUGINS_PATH")
+		)
+
+		AfterEach(func() {
+			if originalPluginPath != "" {
+				// restore the original value
+				err := os.Setenv("KB_PLUGINS_PATH", originalPluginPath)
+				Expect(err).To(BeNil())
+			} else {
+				// unset if it was originally unset
+				err := os.Unsetenv("KB_PLUGINS_PATH")
+				Expect(err).To(BeNil())
+			}
```

Comment:
We should do this logic for _all_ the environment variables we modify during testing. It looks like the only one we are probably missing here is doing this logic for the `XDG_CONFIG_HOME` variable